code/game/objects/items/devices/transfer_valve.dm

```diff
@@ -11,6 +11,12 @@
 	var/toggle = 1
 	flags = PROXMOVE
 
+	get_tax()

```

Comment:
relative pathing bad

Reply:
b-but goon does it and gets away with it